content/docs/user-guide/components/reference/atom/sky-atmosphere.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Sky Atmosphere Component
+linktitle: Sky Atmosphere
+description: 'Open 3D Engine (O3DE) Sky Atmosphere component reference.'
+toc: true
+---
+
+The **Sky Atmosphere** component provides physically-based sky and atmosphere rendering capabilities.
+
+The sky is rendered by sampling points on rays that start at the camera and intersect with the atmosphere.  Because sampling many rays every frame is an expensive operation, luminence and volume look up tables are used to perform fewer calculations.  
+
+![Sky atmosphere in desert mountain scene](/images/user-guide/components/reference/atom/sky-atmosphere/desert-day.jpg)
+
+## Provider
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
```

Comment:
although this is not the developer guide, perhaps an extra blurb describing how atom provides for the sky atmosphere could make it easier to understand how to work with it?